gluten-ut/spark33/src/test/scala/io/glutenproject/utils/velox/VeloxTestSettings.scala

```diff
@@ -18,12 +18,14 @@
 package io.glutenproject.utils.velox
 
 import io.glutenproject.utils.BackendTestSettings
-
+import org.apache.spark.sql.catalyst.expressions.GlutenMathExpressionsSuite
 import org.apache.spark.sql.{GlutenBloomFilterAggregateQuerySuite, GlutenStringFunctionsSuite}
 
 class VeloxTestSettings extends BackendTestSettings {
   enableSuite[GlutenStringFunctionsSuite]
   enableSuite[GlutenBloomFilterAggregateQuerySuite]
     // fallback might_contain, the input argument binary is not same with vanilla spark
     .exclude("Test NULL inputs for might_contain")
+  enableSuite[GlutenMathExpressionsSuite]
+    .include("sec", "csc")
```

Comment:
This PR aims to support `asinh, acosh, atanh, sec, csc` functions. Is there any blocking issue for the other functions?

Reply:
asinh, acosh, atanh will be tested in spark32, sec and csc were added since Spark 3.3.0, and I'm not sure other tests of MathExpressionSuite could be passed in spark33, so I only include sec and csc.